data/org.freedesktop.portal.Documents.xml

```diff
@@ -273,5 +273,17 @@
       <arg type='s' name='app_id' direction='in'/>
       <arg type='a{say}' name='docs' direction='out'/>
     </method>
+
+    <!--
+        GetRealPath:
+        @doc_id: the ID of the file in the document store
+        @path: the path for the file in the host filesystem
+
+        Gets the filesystem path for a document store entry.
```

Comment:
Following some other methods described here in this XML file, I think it should mention that this call is available inside the sandbox when the app has access to the given file. Also a version bump for this portal might be needed so clients can check whether this method is available or not.

Reply:
Please, also mention that this method was introduced in version 5 of the document portal. Similar to [other methods](https://github.com/flatpak/xdg-desktop-portal/blob/main/data/org.freedesktop.portal.Documents.xml#L130).